### PR TITLE
docs: update apisix_upstream.md

### PR DESCRIPTION
**Pull request by wangyp0701**
Update description of scheme

### Type of change:

- [x] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix-ingress-controller#community) first**

### CODE REVIEW

**Comment by Chever-John** [comment]
Could you mention https://github.com/apache/apisix-ingress-controller/issues/982 ?

File: docs/en/latest/references/apisix_upstream.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ title: ApisixUpstream Reference
 
 |     Field     |  Type    | Description    |
 |---------------|----------|----------------|
-| scheme        | string   | The protocol used to talk to the Service, can be `http`, `grpc`, default is `http`.   |
+| scheme        | string   | The protocol used to talk to the Service, can be  `http`, `https`, `grpc`, `grpcs` default is `http`.   |
```
Comment (Chever-John) [inline docs/en/latest/references/apisix_upstream.md:26]:
```suggestion
| scheme        | string   | The protocol used to talk to the Service, can be `http`, `https`, `grpc`, `grpcs` default is `http`.   |
```
There is an extra space here.

**Review by tao12345666333 — APPROVED** [review]
LGTM

Thanks

**Review by juzhiyuan — APPROVED** [review]

**Comment by lijing-21** [comment]
Hi @wangyp0701  , thanks for the contribution!

Here is the Contributor T-shirt form[1], if you're interested, kindly take a look :)

[1] https://github.com/apache/apisix/blob/master/CONTRIBUTING.md#contributor-t-shirt

**Pull request merged**